internal/fetchraw/compliancescans.go

```diff
@@ -308,6 +311,17 @@ func getPVCExtractorPod(objName, ns, image, claimName string) *corev1.Pod {
 					Name:    "pv-extract-pod",
 					Image:   image,
 					Command: []string{"sleep", "inf"},
+					SecurityContext: &corev1.SecurityContext{
+						Capabilities: &corev1.Capabilities{
+							Drop: []corev1.Capability{"ALL"},
+						},
+						Privileged:               &bFalse,
+						RunAsNonRoot:             &bTrue,
```

Comment:
Based on the variables defined above, `RunAsNonRoot` is evaluating to `false`. Is that what we want?

Reply:
no you're right